__tests__/util/index.js

```diff
@@ -1,16 +1,8 @@
-const child_process = require('child_process') // eslint-disable-line camelcase
-const fs = require('fs')
+const fs = require('fs-extra')
 const os = require('os')
 const path = require('path')
 
-const lockfile = require('proper-lockfile')
-const tar = require('tar')
-
-const { createReleaseArchive, createReleaseArchiveSync } = require('../../internal_lib/create-release-archive')
-
-const repoDir = path.resolve(__dirname, '..', '..')
-
-var _worktreeCommit = process.env.KIT_JEST_WORKTREE_COMMIT || undefined
+const child_process = require('child_process') // eslint-disable-line camelcase
```

Comment:
I'd suggest leaving this line where it was to keep the diff as straightforward as possible.